### ISSUE TITLE
Assign H2 to Textbox Headers

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/ideas#499]
### Feature Description

As a user, I would like to have textbox headers assigned a H2 heading by default.

### Feature Use Case

From a network manager based on review by their Digital Accessibility Team: We are finding that a lot of our authors are using textboxes, which we love and believe is a great pedagogical and organizational tool, but most leave the text in the header box at paragraph level when many times it should be changed to H2.

[Link to ticket](https://pressbooks.zendesk.com/agent/tickets/21386) (for internal reference)

**Comment by SteelWagstaff** [comment]
Related: https://github.com/pressbooks/ideas/issues/409. @cmurtagh this is a ticket I'd be happy to open a PR for if the team was open to it?

**Comment by cmurtagh** [comment]
Let's confirm with @greatislander that this is indeed the best a11y way to do this. If yes, then please do send a PR.

**Comment by MichSpieler** [comment]
A different approach has been suggested in https://github.com/pressbooks/private/issues/1769, which is to add a region role. This might be the better approach as `<h2>` may often not be the correct heading level.

**Comment by SteelWagstaff** [comment]
@MichSpieler can we close this as `won't fix` and defer to the recommended approach in https://github.com/pressbooks/private/issues/1769#issuecomment-4605870181?

**Comment by greatislander** [comment]
Good find @SteelWagstaff— I'd suggest closing this as a duplicate of pressbooks/private#1769.
